src/main/java/com/hms_networks/americas/sc/extensions/datapoint/DataPoint.java

```diff
@@ -133,9 +133,16 @@ public String toString() {
   public abstract DataType getType();
 
   /**
-   * Get the string representation of the data point value.
+   * Get the {@link String} representation of the data point value.
```

Comment:
This belongs in a separate commit

Reply:
Changed (and covered the instances of this in subclasses)